### PR TITLE
MNT: sort out the xspress3 handlers

### PR DESCRIPTION
**Pull request by tacaswell**

### CODE REVIEW

File: area_detector_handlers/tests/test_xs3.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import pkg_resources
+import pytest
+
+
+def select_handler(spec):
+    handlers = [
+        ep.load()
+        for ep in pkg_resources.iter_entry_points("databroker.handlers")
```
Comment (danielballan) [inline area_detector_handlers/tests/test_xs3.py]:
```suggestion
        for ep in entrypoints.get_group_named("databroker.handlers").values()
```

File: area_detector_handlers/tests/test_xs3.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import pkg_resources
```
Comment (danielballan) [inline area_detector_handlers/tests/test_xs3.py]: The implementation in `pkg_resources` is "problematic", per Min. I think it's something about being slow and maybe bad with eggs. Main reason to care here is just to set a good example / be consistent in using `entrypoints` everywhere.

```suggestion
import entrypoints
```

File: area_detector_handlers/tests/test_entrypoints.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import pkg_resources
```
Comment (danielballan) [inline area_detector_handlers/tests/test_entrypoints.py]:
```suggestion
import entrypoints
```

File: area_detector_handlers/tests/test_entrypoints.py [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
+
+def test_entrypoint_consisetncy():
+    for ep in pkg_resources.iter_entry_points("databroker.handlers"):
```
Comment (danielballan) [inline area_detector_handlers/tests/test_entrypoints.py]:
```suggestion
    for ep in entrypoints.get_group_named("databroker.handlers").values():
```

File: area_detector_handlers/spe_reader.py [hunk 5]
```diff
@@ -17,18 +17,15 @@
 #
 # Part of the "pyspec" package
 #
-from __future__ import (absolute_import, division, print_function,
-                        unicode_literals)
-import six
-
+# This is heavily modified from the original
```
Comment (danielballan) [inline area_detector_handlers/spe_reader.py]:
```suggestion
# This is heavily modified from the original in
# https://github.com/stuwilkins/pyspec/blob/master/pyspec/ccd/files.py
# adding py3 support and changing the signatures.
```

File: area_detector_handlers/tests/test_xs3.py (hunk 2 above)
Comment (danielballan) [reply area_detector_handlers/tests/test_xs3.py]: Explanation: https://github.com/takluyver/entrypoints/pull/37/files

File: setup.py [hunk 6]
```diff
@@ -40,21 +40,22 @@
     description="DataBroker 'handlers' for Area Detector",
     long_description=readme,
     author="Brookhaven National Lab",
-    author_email='dallan@bnl.gov',
-    url='https://github.com/danielballan/area-detector-handlers',
-    python_requires='>={}'.format('.'.join(str(n) for n in min_version)),
-    packages=find_packages(exclude=['docs', 'tests']),
+    author_email="dallan@bnl.gov",
+    url="https://github.com/danielballan/area-detector-handlers",
```
[the remark this answers is not in the document]
Comment (mrakitin) [reply setup.py]: It's still showing https://github.com/danielballan/area-detector-handlers.

**Pull request merged**